### PR TITLE
[sailfishos][gecko][embedlite] Bring SDK back in esr78 build. Fixes JB#54390

### PR DESCRIPTION
**Pull request by paveltkv**

### CODE REVIEW

File: rpm/0023-sailfishos-gecko-Force-to-build-mozglue-and-xpcomglu.patch [hunk 1]
```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```
Comment (rainemak) [inline rpm/0023-sailfishos-gecko-Force-to-build-mozglue-and-xpcomglu.patch:1]: This patch looks a bit suspicions actually as sha1 408fd387 was enough for esr60.

**Comment by okodron** [comment]
Previous discussion was here: https://git.sailfishos.org/mer-core/gecko-dev/merge_requests/336

**Comment by rainemak** [comment]
> Previous discussion was here: https://git.sailfishos.org/mer-core/gecko-dev/merge_requests/336

Related amended comment from the previous:

"My gut feeling is that we probably need to do some magic with the sqlite later to make Qt and gecko work nicely hand-in-hand. That said LGTM for now."

**Review by rainemak — APPROVED** [review]

**Comment by rainemak** [comment]
Still LGTM

**Review by okodron — APPROVED** [review]
LGTM

**Pull request merged**